pkg/util/util.go

```diff
@@ -92,3 +94,13 @@ func GetAccessModes(caps []*csi.VolumeCapability) *[]string {
 	}
 	return &modes
 }
+
+func RemoveMultipathDevice(device string) error {
```

Comment:
We should aim to push this code to https://github.com/kubernetes-csi/csi-lib-fc

Reply:
PR Raised: https://github.com/kubernetes-csi/csi-lib-fc/pull/4